html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2><strong>by SnuggieStore</strong></h2>
+    <p>Price: <s>$19.99</s>$14.99</p>
```

Comment:
You wouldn't want `<strong>` here for a couple of reasons.

1. Strong will bold the text, but `header` elements are bold my default
2. It will emphasis this text for screen readers/visual assistance